<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>História:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>600</width>
      <height>347</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sériový port</string>
     </attribute>
     <widget class="QPushButton" name="btn_com_refresh">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>25</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Vyhľadať</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_com_connect">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>25</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pripojiť</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Príkaz:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_command">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_com_send">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Odoslať</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_history">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>322</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Parita:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Stop bity:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Dátové bity:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Rýchlosť:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QComboBox" name="comboBox_ports"/>
       </item>
       <item row="2" column="5">
        <widget class="QComboBox" name="comboBox_speed"/>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="comboBox_parity">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="comboBox_data_bits">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_stop_bits">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QRadioButton" name="radioButton_ascii">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ASCII</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_bin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bin</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_oct">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Oct</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_dec">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dec</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_hex">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hex</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Formát:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_append_crc">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pripojiť CRC16</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_commands">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>---</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
